iidm/iidm-converter-api/src/test/java/com/powsybl/iidm/ConversionParametersTest.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm;
+
+import com.powsybl.iidm.parameters.Parameter;
+import com.powsybl.iidm.parameters.ParameterDefaultValueConfig;
+import com.powsybl.iidm.parameters.ParameterType;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Agnes Leroy <agnes.leroy at rte-france.com>
+ */
+public class ConversionParametersTest {
+    @Test
+    public void testConversionParameters() {
+        Properties properties = new Properties();
+        properties.put("test-param-boolean", "true");
+        Parameter paramBoolean = new Parameter("test-param-boolean", ParameterType.BOOLEAN, "", Boolean.FALSE);
+        Parameter paramBoolean2 = new Parameter("test-param-boolean2", ParameterType.BOOLEAN, "", Boolean.FALSE);
+        assertTrue(ConversionParameters.readBooleanParameter("TEST", properties, paramBoolean, ParameterDefaultValueConfig.INSTANCE));
+        assertFalse(ConversionParameters.readBooleanParameter("TEST", properties, paramBoolean2, ParameterDefaultValueConfig.INSTANCE));
+
+        properties.put("test-param-string", "TestProperty");
+        Parameter paramString = new Parameter("test-param-string", ParameterType.STRING, "", "Test");
+        Parameter paramString2 = new Parameter("test-param-string2", ParameterType.STRING, "", "TestParam");
+        assertEquals("TestProperty", ConversionParameters.readStringParameter("TEST", properties, paramString, ParameterDefaultValueConfig.INSTANCE));
+        assertEquals("TestParam", ConversionParameters.readStringParameter("TEST", properties, paramString2, ParameterDefaultValueConfig.INSTANCE));
+
+        List<String> stringList = new ArrayList<>();
+        stringList.add("Test1Property");
+        stringList.add("Test2Property");
+        List<String> stringList2 = new ArrayList<>();
+        stringList2.add("Test1Param");
+        stringList2.add("Test2Param");
+        properties.put("test-param-string-list", stringList);
+        Parameter paramStringList = new Parameter("test-param-string-list", ParameterType.STRING_LIST, "", stringList);
```

Comment:
Maybe the default value for this parameter should not be `stringList` to prevent false positive in the test?